data/core/init.lua

```diff
@@ -29,6 +29,7 @@ local function save_session()
   if fp then
     fp:write("return {recents=", common.serialize(core.recent_projects),
       ", window=", common.serialize(table.pack(system.get_window_size())),
+      ", window_mode=", common.serialize(system.get_window_mode()),
```

Comment:
Why using serialize here ? The function just returns a string.

Reply:
I mistakenly think it was a function to write to file.

Reply:
The function serialize is needed for tables, to "serialize" them as string. On the other side it is harmless to use it here on a string but if you can correct this I prefer.

Reply:
I changed now locally using `tostring()` but it doesn't work, same as not using anythingm so getting `normal` instead `"normal"`, I'm rusty with lua, which should be the correct way?